File: ProjectManager/ProjectManager/App/Components/Project/Project.js
var Project = function(id, name, description, memberList, creator, pictureUrl, storyList, sprintList){
    this.id = id;
    this.name = name;
    this.description = description;
    this.memberList = memberList;
    this.creator = creator;
    this.pictureUrl = pictureUrl;
    this.storyList = storyList;
    this.sprints = sprintList;
    
    this.createdAt = Date.now();
};

File: ProjectManager/ProjectManager/Models/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProjectManager.Models
{
    public class Project
    {
        public int id { get; set; }
        public String name { get; set; }
        public String description { get; set; }
        public ICollection<User> menberList { get; set; }
        public User creator { get; set; }
        public String pictureUrl { get; set; }
        public ICollection<Story> stories { get; set; } 
        public ICollection<Sprint> sprints{ get; set; }
        public DateTime createdAt { get; set; }

        public Project()
        {
            
        }

        public void addStory(Story story)
        {
            stories.Add(story);
        }

        public void addSprints(Sprint sprint)
        {
            sprints.Add(sprint);
        }

        public void addMember(User user)
        {
            menberList.Add(user);
        }

    }
}